x11-libs/gtk3/gtk3-3.24.30.recipe

```diff
@@ -5,14 +5,14 @@ ranging from small one-off projects to complete application suites."
 HOMEPAGE="https://www.gtk.org"
 COPYRIGHT="GNOME Desktop"
 LICENSE="GNU LGPL v2"
-REVISION="5"
+REVISION="6"
```

Comment:
Not needed.

Reply:
Bumping the revision will cause rebuilds everywhere. Merely enabling the architecture suffices.

Reply:
Done